bin/tx-prover/src/proxy/metrics.rs

```diff
@@ -0,0 +1,81 @@
+use std::sync::LazyLock;
+
+use prometheus::{
+    register_histogram, register_int_counter, register_int_counter_vec, register_int_gauge,
+    Histogram, IntCounter, IntCounterVec, IntGauge,
+};
+
+// QUEUE METRICS
+// ================================================================================================
+
+pub static QUEUE_SIZE: LazyLock<IntGauge> =
+    LazyLock::new(|| register_int_gauge!("queue_size", "Number of requests in the queue").unwrap());
+pub static QUEUE_LATENCY: LazyLock<Histogram> = LazyLock::new(|| {
+    register_histogram!(
+        "queue_latency",
+        "Time requests spend in the queue",
+        vec![0.1, 0.5, 1.0, 2.0, 5.0, 10.0]
+    )
+    .unwrap()
+});
+pub static QUEUE_DROP_COUNT: LazyLock<IntCounter> = LazyLock::new(|| {
+    register_int_counter!("queue_drop_count", "Number of requests dropped due to a full queue")
+        .unwrap()
+});
+
+// WORKER METRICS
+// ================================================================================================
+
+pub static WORKER_COUNT: LazyLock<IntGauge> =
+    LazyLock::new(|| register_int_gauge!("worker_count", "Number of workers").unwrap());
+pub static WORKER_UNHEALTHY: LazyLock<IntCounter> = LazyLock::new(|| {
+    register_int_counter!("worker_unhealthy", "Number of unhealthy workers").unwrap()
+});
```

Comment:
Could this number decrease (i.e. an unhealthy worker is fixed)? If so, this should be an `IntGauge`.

Reply:
It is meant to keep track of how many workers failed. It should not decrease since it doesn't keep track of **which** worker failed.

Reply:
@igamigo answering to:

> Considering this, should the description change a bit? Something like "Number of times that workers were registered as unhealthy"

Yes, makes sense.